python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -641,6 +641,10 @@ def _(node: pl_expr.Literal, translator: Translator, dtype: plc.DataType) -> exp
         return expr.LiteralColumn(
             dtype, data.cast(dtypes.downcast_arrow_lists(data.type))
         )
+    if dtype.id() == plc.TypeId.LIST:  # pragma: no cover
```

Comment:
I think with https://github.com/rapidsai/cudf/pull/18564 this will be unneeded, but we'll just need to make sure the arrow interop does this conversion correctly